src/components/BookCard/BookCard.jsx

```diff
@@ -0,0 +1,32 @@
+import BookList from "../../containers/BookList/BookList";
+import styles from "./BookCard.module.scss";
+
+const BookCard = ({ book }) => {
+    return (
+        <div className={styles.BookCard}>
+            <img
+                className={styles.BookCard__img}
+                src={
+                    book.volumeInfo.imageLinks
+                        ? book.volumeInfo.imageLinks.thumbnail
+                        : "https://picsum.photos/150/200"
```

Comment:
Good work identifying that this needs a default case. It's probably better to decouple this logic from the presentational component though. Another approach would be having a function designed to tidy up book data.